app/models/schedule.py

```diff
@@ -0,0 +1,162 @@
+from abc import ABC, abstractmethod
+from calendar import monthrange
+from app.models.availability import Availability
+from app.helpers.date import Date
+
+
+class Schedule(ABC):
+
+    def __init__(self):
+
+        pass
+
+    @abstractmethod
+    def get_schedule_date_string(self):
+        """
+
+        :return: Should return some date string representing
+         the schedule's representation scope (month or day) for front-end display
+        """
+        pass
+
+class DailySchedule(Schedule):
```

Comment:
Why not put these in their own file/module?
schedule.py
daily_schedule.py
monthly_schedule.py

Reply:
I don't mind having them in the same file

Reply:
I just remember in 343 we did the same thing and the file in the end was a huge mess, but if we feel like that won't happen we can keep it this way

Reply:
it's only for the three schedule classes, otherwise i'd agree w/ you